scripts/run-all-universal.sh

```
@@ -0,0 +1,168 @@
+      --env GHOSTFOLIO_UPDATE_CASH="${GHOSTFOLIO_UPDATE_CASH:-TRUE}" \
+      --env GHOSTFOLIO_URL="$GHOSTFOLIO_URL" \
+      --env GHOSTFOLIO_SECRET="$GHOSTFOLIO_SECRET" \
+      --env NODE_OPTIONS="${NODE_OPTIONS:---max-old-space-size=4000}" \
+      --add-host=host.docker.internal:host-gateway \
+      dickwolff/export-to-ghostfolio
+    
+    # Allow time for file system sync
+    sleep 20
+    
+    # Locate the generated JSON
+    latest_json=$(ls -t out/ghostfolio-*.json 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)